src/rules/style.ts

```diff
@@ -43,6 +43,8 @@ export default {
   'no-consecutive-blank-lines': [true, 2],
   // Disallows parameter properties in class constructors.
   'no-parameter-properties': false,
+  // Don’t if you import “foo” anyway.
```

Comment:
this sentence doesn't seem to make sense? As a result I don't understand the rule